.github/workflows/build.yml

```diff
@@ -25,25 +25,32 @@ jobs:
     strategy:
       matrix:
         rust:
-          - stable
-          - beta
+          # XXX TBD SKIP FOR NOW
+          # - stable
+          # - beta
           - nightly
         os:
-          - ubuntu-latest
-          - windows-latest
+          # XXX TBD SKIP FOR NOW TO SPEED UP CI BUILD TESTING
+          # - ubuntu-latest
+          # XXX TBD SKIP FOR NOW
+          # - windows-latest
           - macos-latest
         exclude:
+          # XXX TBD ??? ???
           # only stable on macos/windows (slower platforms)
-          - os: windows-latest
-            rust: beta
-          - os: windows-latest
-            rust: nightly
-          - os: macos-latest
-            rust: beta
-          - os: macos-latest
-            rust: nightly
+          # XXX TBD ??? ???
+          # - os: windows-latest
+          #   rust: beta
+          # - os: windows-latest
+          #   rust: nightly
+          # - os: macos-latest
+          #   rust: beta
+          # - os: macos-latest
+          #   rust: nightly
```

Comment:
should update these to favor testing with Rust nightly when re-enabled to improve consistency with coverage job update